### PR TITLE
Background cards

### PR DESCRIPTION
**Pull request by dorpvom**
Introduced borderless cards with light background as primary way of ordering elements.
Applied this to home and statistics pages

### CODE REVIEW

File: src/web_interface/templates/macros.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{% macro start_stats_panel(panel_title, icon="chart-bar") -%}
+    <!-- stats: {{ panel_title }} -->
+    <div class="col-sm-6 col-md-4 mb-3 px-2">
+        <div class="card p-1 bg-light rounded border-0">
+            <div class="card-body bg-light">
+                <h5 class="card-title mb-3"><i class="fas fa-{{ icon }}"></i> {{ panel_title }}</h5>
+{%- endmacro %}
+
+{% macro end_stats_panel(counter) %}
```
Comment (jstucke) [inline src/web_interface/templates/macros.html:9]: this seems not very elegant but there is a better way: [jinja calls](https://jinja.palletsprojects.com/en/2.11.x/templates/#call)
```html
{% macro stats_panel(panel_title, icon="chart-bar") -%}
    <!-- stats: {{ panel_title }} -->
    <div class="col-sm-6 col-md-4 mb-3 px-2">
        <div class="card p-1 bg-light rounded border-0">
            <div class="card-body bg-light">
                <h5 class="card-title mb-3"><i class="fas fa-{{ icon }}"></i> {{ panel_title }}</h5>
                {{ caller() }}
            </div>
        </div>
    </div>
{% endmacro %}
```

and in `show_statistics.html`:
```html
{% call macros.stats_panel("General") %}
    <table (...)/>
{% endcall %}
```

File: src/web_interface/templates/show_statistic.html [hunk 2]
```diff
@@ -59,7 +59,7 @@ <h3 class="mb-3">Firmware Statistics</h3>
 
  	{% if stats["general_stats"] %}
         {{ macros.start_stats_panel("General") }}
```
Comment (jstucke) [inline src/web_interface/templates/show_statistic.html:61]: jinja calls (see above)

File: src/web_interface/templates/database/database_browse.html [hunk 3]
```diff
@@ -73,7 +73,7 @@ <h3 class="mb-3">Browse Firmware Database</h3>
 <div class="row justify-content-center">
     <div class="mt-4 col-xl-6 col-lg-8 col-md-10">
         {% if firmware_list %}
-        <ul class="list-group">
```
Comment (jstucke) [inline src/web_interface/templates/database/database_browse.html:76]: there is still a (broken) border around the "Show Search Query" element
class `border-0` seems to fix the lower part but not the button
a style `border: 0;` seems to do the trick, though

File: src/web_interface/templates/macros.html (hunk 1 above)
Comment (dorpvom) [reply src/web_interface/templates/macros.html:9]: I have to admit, I knew about this feature but I didn't really understand it

File: src/web_interface/templates/database/database_browse.html (hunk 3 above)
Comment (dorpvom) [reply src/web_interface/templates/database/database_browse.html:76]: New version checked in, including fix of quotation

File: src/helperFunctions/web_interface.py [hunk 4]
```diff
@@ -14,12 +14,16 @@
...
 
 def get_color_list(number, limit=15):
-    compliant_colors = ['#2b669a', '#cce0dc', '#2b669a', '#cce0dc', '#2b669a', '#cce0dc',
-                        '#2b669a', '#cce0dc', '#2b669a', '#cce0dc', '#2b669a', '#cce0dc',
-                        '#2b669a', '#cce0dc', '#2b669a', '#cce0dc', '#2b669a', '#cce0dc']
-    return compliant_colors[:number if number <= limit else limit]
+    return [
+        BS_PRIMARY, BS_SECONDARY, BS_PRIMARY, BS_SECONDARY, BS_PRIMARY, BS_SECONDARY,
+        BS_PRIMARY, BS_SECONDARY, BS_PRIMARY, BS_SECONDARY, BS_PRIMARY, BS_SECONDARY,
+        BS_PRIMARY, BS_SECONDARY, BS_PRIMARY, BS_SECONDARY, BS_PRIMARY, BS_SECONDARY
```
[the remark this answers is not in the document]
Comment (dorpvom) [reply src/helperFunctions/web_interface.py]: Brackets did the trick

**Pull request merged**